rest_framework_json_api/utils.py

```diff
@@ -320,7 +320,10 @@ def get_resource_type_from_serializer(serializer):
         return meta.resource_name
     elif hasattr(meta, "model"):
         return get_resource_type_from_model(meta.model)
-    raise AttributeError()
+    # inspect.currentframe().f_code.co_firstlineno
+    raise AttributeError(
+        f"can not detect 'resource_name' on serializer '{serializer.__class__.__name__}'" 
+        f" in module '{serializer.__class__.__module__}:{inspect.getsourcelines(serializer.__class__)[-1]}'")
```

Comment:
Sometimes this attribute error is simply ignored (see e.g. [here](https://github.com/django-json-api/django-rest-framework-json-api/blob/master/rest_framework_json_api/utils.py#L65))

For this let's not add source lines as it might have a performance impact.